src/values.rs

```diff
@@ -8,62 +8,143 @@ use bevy_transform::prelude::Transform;
 use rand::seq::SliceRandom;
 use rand::{prelude::ThreadRng, Rng};
 
+/// Describes an oriengted segment of a circle with a given radius.
+#[derive(Debug, Clone, Reflect, FromReflect)]
+pub struct CircleSegment {
+    /// The shape of the emitter, defined in radian.
+    ///
+    /// The default is [`std::f32::consts::TAU`], which results particles going in all directions in a circle.
```

Comment:
```suggestion
    /// The default is [`std::f32::consts::TAU`], which results particles going in all directions in a circle.
```

perhaps it's better to say 2*PI, not everyone might know what tau is?

Reply:
```suggestion
    /// The default is `2 * PI`, which results particles going in all directions in a circle.
```

Ah yeah, fair point.